types/pino/index.d.ts

```diff
@@ -13,7 +13,7 @@
 import * as stream from 'stream';
 import * as http from 'http';
 import { EventEmitter } from 'events';
-import SonicBoom = require('sonic-boom');
```

Comment:
Sorry for the confusion! I just saw that `sonic-boom` is a CommonJS module (see my comment below) and **your** initial import was correct: `import SonicBoom = require('sonic-boom');` but the other ones above were wrong. Could you change the other imports to `import ... = require()` as well? Many thanks 👍

Reply:
Sure! Should be fixed now